test/utils.js

```diff
@@ -1,3 +1,25 @@
+const BlockMinerContract = artifacts.require("./BlockMiner.sol");
+
+class BlockMiner {
+  async init() {
+    this.blockMiner = await BlockMinerContract.deployed();
+  }
+
+  async mine(numberOfBlocks) {
+    for(let i = 0; i < numberOfBlocks; i++) {
+      await this.blockMiner.mine();
+    }
+  }
+
+  async mineUntilBeginningOfNextRound(roundLength) {
```

Comment:
Do we care that rounds can skip? Ex: `initializeRound` called at blocks 2 and 8 with roundLength of 3 - This would start us at round 0 and continue with round 2.

Reply:
Livepeer' specs says it's fine. And it provides for free the information of when each round happened

Reply:
Sounds good.